src/Appwrite/Utopia/Response/Model/Attribute.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class Attribute extends Model
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('$id', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Attribute ID.',
+                'default' => '',
+                'example' => '',
+            ])
+            ->addRule('type', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Attribute type.',
+                'default' => '',
+                'example' => '',
+            ])
+            ->addRule('size', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Attribute size.',
+                'default' => 0,
+                'example' => 0,
+            ])
+            ->addRule('required', [
+                'type' => self::TYPE_BOOLEAN,
+                'description' => 'Is attribute required?',
+                'default' => false,
+                'example' => false,
+            ])
+            ->addRule('signed', [
+                'type' => self::TYPE_BOOLEAN,
+                'description' => 'Is attribute signed?',
+                'default' => true,
+                'example' => true,
+            ])
+            ->addRule('array', [
+                'type' => self::TYPE_BOOLEAN,
+                'description' => 'Is attribute an array?',
```

Comment:
Maybe we can use the same names for this model in the request params descriptions - instead of the word flag.